src/game_updater/GameUpdater.cpp

```diff
@@ -10,8 +10,9 @@
 #include "TimingConstants.h"
```

Comment:
Some comment blocks before each type of object is modified might make this easier to read and scan through to make changes later.